samples/org.wso2.carbon.config.samples.osgi/pom.xml

```diff
@@ -52,8 +52,8 @@
         </export.package>
         <import.package>
             org.osgi.framework.*;version="${osgi.framework.package.import.version.range}",
-            org.wso2.carbon.secvault.securevault.*,
-            org.wso2.carbon.config.*,
+            org.wso2.carbon.secvault.component.*;version="${carbon.securevault.version.range}",
+            org.wso2.carbon.config.*;version="${carbon.config.package.import.version.range}",
```

Comment:
This OSGi sample will be removed. There will only be a standalone configuration sample.